### PR TITLE
User Data updates

### PR DESCRIPTION
**Pull request by willwatters**

### CODE REVIEW

File: groups/chips-uam-infrastructure/templates/chips_uam_user_data.tpl [hunk 1]
```diff
@@ -25,14 +25,15 @@ rm -rf /home/ec2-user/webswing-2.5.5-distribution.zip
...
 ${CHIPS_UAM_INPUTS}
 EOF
 
-chmod 700 /home/ec2-user/uam/master.txt
+chmod 700 /home/ec2-user/uam_gui-1.109.0-rc1/master.txt
 
 #Encrypt master.txt
-/home/ec2-user/uam/encryptUamDbConfigFile.sh master.txt
+cd /home/ec2-user/uam_gui-1.109.0-rc1/
+/home/ec2-user/uam_gui-1.109.0-rc1/encryptUamDbConfigFile.sh master.txt
```
Comment (MartyFox) [inline groups/chips-uam-infrastructure/templates/chips_uam_user_data.tpl:38]: Add in a remove command to remove the unencrypted master.txt file.

File: groups/chips-uam-infrastructure/templates/chips_uam_user_data.tpl [hunk 2]
```diff
@@ -13,10 +13,10 @@ yum -y install java-1.8.0-openjdk
 yum -y install Xvfb
 
 #UAM GUI
-mkdir -p /home/ec2-user/uam
-aws s3 cp s3://shared-services.eu-west-2.resources.ch.gov.uk/chips/uam/uam_gui-1.109.0-rc1.zip /home/ec2-user/uam
-unzip /home/ec2-user/uam/uam_gui-1.109.0-rc1.zip
-rm -rf /home/ec2-user/uam/uam_gui-1.109.0-rc1.zip
+cd /home/ec2-user/
```
Comment (MartyFox) [inline groups/chips-uam-infrastructure/templates/chips_uam_user_data.tpl]: Check out pushd/popd for changing directories in shell:
https://unix.stackexchange.com/questions/77077/how-do-i-use-pushd-and-popd-commands

Comment (willwatters) [reply groups/chips-uam-infrastructure/templates/chips_uam_user_data.tpl]: pushd applied and tested.

File: groups/chips-uam-infrastructure/templates/chips_uam_user_data.tpl (hunk 1 above)
Comment (willwatters) [reply groups/chips-uam-infrastructure/templates/chips_uam_user_data.tpl:38]: command to remove unencrypted master.txt:

rm -rf /home/ec2-user/uam_gui-1.109.0-rc1/master.txt

**Review by MartyFox — APPROVED** [review]

**Pull request merged**